app/models/story.rb

```diff
@@ -1,5 +1,30 @@
+require 'redcarpet'
+
 class Story < ApplicationRecord
 	belongs_to :user
+	has_many :taggings
+	has_many :tags, through: :taggings
+
+	after_save :save_content_as_markdown
```

Comment:
Nice. But even better - add if statemant that will trigger this hook only when certain filed has changed